src/drawer/index.tsx

```diff
@@ -100,6 +104,24 @@ export default class Drawer extends PureComponent <DrawerProps, DrawerState> {
     this.contentNode = contentNode
   }
 
+  private scrollPosition: number = 0
+
+  scrollReset = () => {
+    // store current scroll position
+    this.scrollPosition = document.documentElement.scrollTop
+    // scroll to top
+    document.documentElement.scrollTop = 0
+    // block html scroll
+    document.querySelector('html').style.overflow = 'hidden'
```

Comment:
I thought you'd need to do it on both: `html` and `body`. Have you checked on multiple browsers?